src/main/java/com/k3ntako/HTTPServer/controllers/SimplePost.java

```diff
@@ -8,17 +8,32 @@
 
 public class SimplePost implements ControllerInterface {
```

Comment:
Let's rename this class to `Reminders` since this will be the controller for all incoming actions(Create, Read, Update, Destroy).